lab-davide/server.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+
+//npm modules//
+const debug =require('debug')('note: server');
```

Comment:
You may want to clean up your formatting here, just for readability and consistency. I like that you're bringing in npm modules separately from the modules you wrote though!